apis/train.py

```diff
@@ -1,8 +1,69 @@
+import os
+import pickle
+import pandas as pd
+
+from numpy.random import RandomState
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import roc_auc_score
+from datetime import datetime
 from flask_restplus import Namespace, Resource, fields
 
 api = Namespace('train', description='Namespace for training')
 
 @api.route('/fit')
 class Train(Resource):
+
     def get(self):
-        return {'success': 'Training pipeline successful'}
\ No newline at end of file
+        self.train()
+        return self.train() # {'success': 'Training pipeline successful',}
+
+    def train(self, file=None, target_var=None):
+
+        # curretnly it's having nothing as i/p
+        start_time = datetime.now()
+        MODELS_ROOT = "./apis/models" # will be changed init
```

Comment:
Let's make dir at `./core/models` and store it there. apis directory is specified to namespaces and api methods only

Reply:
okay